.eslintrc

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": [
+    "@nuxtjs/eslint-config-typescript"
+  ],
+  "rules": {
+    "@typescript-eslint/no-unused-vars": 0,
```

Comment:
I don't think we should complicate the demo setup with a non-standard eslint setup. it should be as simple, so people can see only relevant code. there should be no other un-relevant code in here if possible, I'd be fine with some very simple pionter/additions of nuxt eslint setup, but this is not the point to add any custom configs

Reply:
the setup was changed recently to the new flat config in kickstart. as per your suggestion, let's implement the new setup in nuxtjs-drupal-ce first, then on nuxt3-demo as follow-ups?